pareto/strategic_water_management/run_strategic_model.py

```diff
@@ -220,4 +111,5 @@
 set_list = []
```

Comment:
I believe this should not be here as I do not see why the get_data() function is called at the end of the file again. Should it not be read before hand and the create a model and solve it or this is done on purpose?

Reply:
get_data() was developed as a flexible function to read in both PARETO input AND PARETO output. These lines at the end of this file demonstrate how get_data() can be used to read PARETO output, if desired. This requires passing in custom set_list and custom parameter_list. 
This example was not introduced in this PR. If it's something we'd like removed, let's plan for it in a separate PR!

Reply:
I believe this is the reason for the warnings that a user would get for the invalid pareto tabs. I am not sure if this should be read or not.

Reply:
Yes, this is the exact line that causes the warnings - I think the current version of main also does something similar? With the code you introduced, a warning is raised for unused tabs which would cause similar warning with reading the PARETO output.

Reply:
Yes. I had missed that there. I thought that was some other warning, but with a more comprehensive warning message figured that it is the reason. I am not sure if we should have this in main. Not exactly sure about what is it doing there? Probably @tarnold17 can answer this question.

Reply:
It's existed as long as I have been on the project. I think the run file used to be more of a tutorial than a true run file like it has evolved into now. Maybe it's not worth to have here since we have documentation and jupyter notebooks to demonstrate. I'm OK with removing- @tarnold17 , thoughts?

Reply:
Per our conversation, removed!